google/cloud/pubsublite/internal/partition_publisher.cc

```diff
@@ -176,7 +176,7 @@ void PartitionPublisher::OnRead(absl::optional<PublishResponse> response) {
               return a.start_index() < b.start_index();
             });
   int range_idx = 0;
-  for (int msg_idx = 0; msg_idx < batch.size(); ++msg_idx) {
+  for (int msg_idx = 0; msg_idx < static_cast<int>(batch.size()); ++msg_idx) {
```

Comment:
Please do the opposite:

```cc
for (std::size_t msg_idx = 0; msg_idx != batch.size(); ++msg_idx) {
```

Reply:
I did that initially and there were more errors because `ranges->Get(range_idx)` accepts an int. This requires the least number of static_casts.

Reply:
Ack.